solidity-v1/dashboard/src/connectors/tally-injected.js

```diff
@@ -0,0 +1,10 @@
+import { WALLETS } from "../constants/constants"
+import { InjectedConnector } from "./injected-connector"
+
+class TallyInjectedConnector extends InjectedConnector {
+  constructor() {
+    super(WALLETS.TALLY.name, window.tally)
```

Comment:
@greg-nagy  I might be wrong, but it looks like something with tally extension is not working right, because the tally provider is not injected after refreshing the page 🤔 It works fine when i rebuild the project, but when i manually refresh the page the `window.tally` is `undefined`.

There is also strange behaviour when i disconnect from Tally and connect to it again - it will automatically refresh the page.

Reply:
@michalsmiarowski thanks, I noticed the same. Will get back with fixes soon.

Reply:
We have made some changes so I don't think this will be a problem. But let's test again today.